### PR TITLE
Small code review

### PR DESCRIPTION
**Pull request by ctoffanin**
In this PR:
- Added the `fork` step and fixed the `git clone` command URL in the `readme` page
- Updated `.gitignore` file
-  Renamed folders and components names:
   - It's good practice to use camel case (HomePage) or kebab case (home-page) for the folders name, Spaces should be avoided
   - It's good practice to name the component with the capital letter and use camel case (Button, HomePage)
   - Avoid putting numbers in classes and components' names (Information rather than Information1)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,4 +34,9 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
-next-env.d.ts
+
+# VS code
+.vscode
+
+# IDEA
+.idea
```
Comment (avie-dev) [inline .gitignore]: I'm adding a new line here.

**Review by avie-dev — APPROVED** [review]
I fixed one line and the others seem good. LGTM🌹

**Comment by ctoffanin** [comment]
Don't merge this yet, let others take a look and ask questions if there are doubts.

**Comment by avie-dev** [comment]
@ctoffani Thank you for this Cristian. I see that you applied a rule on the namespace(naming objects). Can you create a rule for this and put it somewhere on Wiki or Readme? and update the current working members?

**Comment by avie-dev** [comment]
@ctoffanin 
> Don't merge this yet, let others take a look and ask questions if there are doubts.

I created a revert PR just in case they have something to change.

**Pull request merged**